packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskInputField/components/TaskInputLabel/TaskInputLabel.js

```diff
@@ -30,7 +32,6 @@ export const StyledTaskInputLabelWrapper = styled(Box)`
   }
 
   img:only-child, svg:only-child {
-    background-color: ${zooTheme.global.colors.brand};
```

Comment:
This shouldn't be removed. This is the background color for projects that are including a single image icon in their markdown, like galaxy zoo.

Reply:
Is it ok if I move the `background-color` styling specifically to the single and multiple tasks in [Commit 4f47](https://github.com/zooniverse/front-end-monorepo/pull/913/commits/4f47250143588bd65881e521d83b2afa13cdc9c3)?

staging project/workflow `markb-panoptes/fem-drawing-tools/classify?workflow=3344` has question task with icons in label.

Reply:
Yes, that works for me.